### PR TITLE
Remove old key, add new key, revise travis decrypt

### PR DESCRIPTION
**Pull request by ahayden**
This PR deploys a new ssh key to the repo that can be decrypted and used by the travis job.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@ branches:
   - prod
 
 before_install:
-  - openssl aes-256-cbc -K $encrypted_7dd696d233c6_key -iv $encrypted_7dd696d233c6_iv -in toptal_org-sagebase-scicomp.pem.enc -out ~/.ssh/toptal_org-sagebase-scicomp.pem -d
+  - openssl aes-256-cbc -K $encrypted_8b76707d972e_key -iv $encrypted_8b76707d972e_iv -in agora-travis.pem.enc -out agora-travis.pem -d
```
Comment (zaro0508) [inline .travis.yml:13]: please remember to remove the old *_key and *_iv from travis settings.

**Review by zaro0508 — COMMENTED** [review]
this depends on deploying a new instance so please add a `depends on` in the PR message.

**Comment by ahayden** [comment]
I think the dependency is more like the instance rebuild (https://github.com/Sage-Bionetworks/Agora-infra/pull/82) depends on this one.

**Review by zaro0508 — CHANGES_REQUESTED** [review]
the updatedb.sh file references `toptal_org-sagebase-scicomp.pem` as well.  need to update those to new files.

File: .travis.yml (hunk 1 above)
Comment (zaro0508) [reply .travis.yml:13]: this should be using the agora-access key to access the DB instance

**Review by zaro0508 — CHANGES_REQUESTED** [review]

**Comment by zaro0508** [comment]
replacing this PR with PR #17

**Pull request closed without merging**